doc/in-progress/sui-kiosk.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Sui Kiosk
+---
+
+Sui Kiosk is a primitive, or building block (a module in the Sui Framework), you can use to build a trading platform for digital assets. Sui Kiosk supports adding digital assets that you can store and list for sale to other users. You can also define rules for the kiosk as part of a transfer policy that controls how purchasers can use the asset after purchase. 
```

Comment:
> how purchasers can use the asset after purchase

Not sure that TransferPolicy specifies "how to use". It's defining conditions which needs to be met for the purchase to be successful.